roxie/udplib/udptrs.cpp

```diff
@@ -684,52 +646,41 @@ class CSendManager : implements ISendManager, public CInterface
                     return 0;
 
                 if (udpSnifferEnabled)
-                    send_sniff(true);
-                parent.send_flow->clear_to_send_received(permit.destNodeIndex);
-                UdpReceiverEntry &receiverInfo = parent.receiversTable[permit.destNodeIndex];
-                bool moreRequested;
+                    send_sniff(sniff_t::busy);
+                UdpReceiverEntry &receiverInfo = parent.receiversTable[permit.destNode.getNodeAddress()];
                 unsigned maxPackets;
-                unsigned payload = receiverInfo.sendData(permit, (parent.myNodeIndex == permit.destNodeIndex), bucket, moreRequested, maxPackets);
-                parent.send_flow->send_done(permit.destNodeIndex, moreRequested);
+                unsigned payload = receiverInfo.sendData(permit, bucket, maxPackets);
```

Comment:
maxPackets isn't used - should it be removed?

Reply:
Not new, I think, but yes